corehq/apps/export/forms.py

```diff
@@ -655,24 +655,6 @@ def extra_fields(self):
     def get_form_filter(self):
         raise NotImplementedError
 
-    def get_multimedia_task_kwargs(self, export, download_id, mobile_user_and_group_slugs=None):
-        """These are the kwargs for the Multimedia Download task,
-        specific only to forms.
-        """
-        datespan = self.cleaned_data['date_range']
-        return {
-            'domain': self.domain_object.name,
-            'startdate': datespan.startdate.isoformat(),
-            'enddate': (datespan.enddate + timedelta(days=1)).isoformat(),
-            'app_id': export.app_id,
-            'xmlns': export.xmlns if hasattr(export, 'xmlns') else '',
-            'export_id': export.get_id,
-            'zip_name': 'multimedia-{}'.format(unidecode(export.name)),
-            'user_types': self._get_es_user_types(mobile_user_and_group_slugs),
-            'group': self.data['group'],
-            'download_id': download_id
-        }
```

Comment:
Haha, this is a vintage old-school HQ code pattern - a function or method whose purpose is to set up a large set of args for initiating another function